docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
+<img src="images/EditRefrence2.png" width="600"/>
+<img src="images/EditRefrence3.png" width="600"/>
```

Comment:
<img width="538" alt="ss9" src="https://user-images.githubusercontent.com/60163283/140519747-fb520964-5f04-4080-a717-db26afd43c6d.PNG">
Same issue, there shldnt be any incoming arrow. Make activation bar bigger than sd box.